client/app/components/game-client.js

```diff
@@ -0,0 +1,62 @@
+import Ember from 'ember';
+import GameChannel from 'vilive/utils/game-channel';
+
+export default Ember.Component.extend({
+  init() {
+    this.set('messages', Ember.makeArray());
+    GameChannel.init(this.get('session-manager').currentToken());
+    GameChannel.on('event', payload => {
+      this.putMessage({
+        server: {
```

Comment:
I'm not a fan of this. Maybe you should have two different methods; a `addClientCommand(msg)` and a `addServerMessage(msg)`. It makes it much more explicit. The implementation of these I'm thinking should create the object to be rendered and one of the attributes on the object is the `cssClass`. 

Your template would be simplified, the code more explicit, and the code to add the messages can diverge since the message from the server will probably contain a lot more then the client's command text.